src/capi.rs

```diff
@@ -17,18 +18,24 @@ pub enum DriverInitParamFFI {
 /// This simply calls env_logger::init()
 /// Usually, it's the library consumer who should add this Rust crate dependency,
 /// however, with a C program, we provide this workaround where we provide an API to do just that.
-#[allow(clippy::missing_safety_doc)]
+/// # Safety
 #[no_mangle]
 pub unsafe extern "C" fn microvmi_envlogger_init() {
     env_logger::init();
 }
 
-#[allow(clippy::missing_safety_doc)]
+/// Entrypoint for libmicrovmi
+/// Initializes a specific driver, or all drivers compiled and returns the first one that succeeded
+///
+/// In case of error, init_error will be allocated with the underlying error message.
+/// The init_error pointer should be freed with rs_cstring_free()
+/// # Safety
```

Comment:
Is that an empty Safety section? Why?

Reply:
because there is no additional safety comment to add, but clippy will throw a warning without this section.
maybe i should have disabled the clippy warning then ?

Reply:
Well, the safety section should document the caller's responsibilities in order to uphold safety guarantees, so you should move line 31 into this section, for example.